RFC-010-core-strategy/core-strategy.md

```diff
@@ -0,0 +1,65 @@
+# Executive Summary
+
+The Good Docs Project has grown organically since its creation, and has blossomed into a thriving community. Now it’s important to articulate where we are, how we can best support our users and community members, where we’d like to grow in future. 
+
+
+## Core Strategy Statement:  
+
+_The Good Docs Project educates and empowers people to create high quality documentation by providing them with resources, best practices and tools to enhance their documentation in open source and beyond._
+
+
+## What we do
+
+First and foremost, we create templates with supporting meta-documents that underpin the templates.
+
+As part of this work, we engage in many other initiatives around docs advocacy, docs education, and docs tooling. 
+
+We produce content to address the goals of engaging in our community and helping run the project; and learning and gaining experience in the realm of tech writing.
+
+
+### We don’t compete, we cooperate
+
+We don't compete with other groups and we aren't a business. We are one open source community looking to make open source, as a whole, better for everyone. 
+
+Our project creates templates. We recognize that many folk are creating great templates out in the wild. Some folk are in private enterprise and can’t share their work. 
+Some folk are in open source, and their work isn’t reviewed or validated by others. 
+When we encounter great templates in the wild, we approach the creators to encourage them to join our community and donate their templates for inclusion in our project.
+
+
+## Who we are
+
+We are a group of technical folk. Writers, developers, editors and more. We are passionate about technical writing, documentation and open source.
+
+
+### Our Audience
+
+People in our audience may be one or all of the following:
+
+* Users of our products
+* Members of our community
+* People seeking education and learning opportunities
+
+
+### Our friends
+
+We are friends with (and many of us are active in) the [Write The Docs community](https://www.writethedocs.org/). 
+We are not competing with that community. We acknowledge that our interests overlap and that sometimes our activities align. 
+The relationship between our projects is mutually supportive and friendly.
+
+We welcome sponsorship opportunities. We like to participate in writing programs and open source programs (like the Google Season of Docs).
+
+
+## Differentiation
+
+The Good Docs Project templates undergo a rigorous review process, they are backed by research and they are freely available for anyone to use.
+
+Our key differentiator is: **Best Practice**. 
+
+Down the road, this might logically lead us to becoming a standards organization.
+
+
+## What we don’t do
+
```

Comment:
If we want to be even clearer: what about meetups, local writing days, etc? 
Maybe we're not hosting full-blown conferences but it might be good to know if some smaller versions of a conf (like a meetup) will also be categorically excluded for some reasons (for example we don't want to fragment our community into groups of "seeing each other in person" and "contributing remote only". 

This is just food for thought if we should expand the section on things we don't do.

Reply:
Nice one, thanks @kickoke. I'm keen to hear what others think for this section too.

Reply:
Agreed - love these ideas. Do we even need the 'what we don't do' section? Maybe I just remove it altogether 🤔

Reply:
We might also title this something like "We we aren't trying to do or provide" as a way to highlight that even if we have things like write-ins, those are one-offs and not without the scope of the project.

That said, this seems to escape the scope of "content strategy", doesn't it? It's about the structure above content: the organization.

Reply:
I recommend cutting this section. Even where true, they shouldn't be pinned to a content strategy doc as a source of truth.

Reply:
After reading through it again, I agree with removing this section.

Reply:
Added these comments to the 'feedback' section and removed the What we don't do section of the document.